settings/src/payment.rs

```diff
@@ -107,8 +107,8 @@ pub struct PaymentSettings {
     pub eth_private_key: Option<PrivateKey>,
     // Our own eth Address, derived from the private key on startup and not stored
     pub eth_address: Option<Address>,
-    #[serde(default)]
-    pub balance: Uint256,
+    #[serde(skip_serializing)]
+    pub balance: Option<Uint256>,
```

Comment:
Add the balance as an option here and remove it from the config https://github.com/althea-net/althea_rs/blob/master/rita_common/src/blockchain_oracle/mod.rs#L76

the skip serializing is the right flag and will do what we want. So good call there, but if we don't need to deserialize it ever we should put it somewhere else.